example/src/services/HearthstoneService.js

```diff
@@ -0,0 +1,19 @@
+import { wrapService, withPostSuccess } from 'redux-recompose';
+
+import responseBody from './response';
+
+// Declare your api calls
+const getCards = async () => new Promise(resolve => setTimeout(() => resolve(responseBody), 1000));
+
+// Declare your customizations, used by fetchMiddleware
+getCards.successSelector = response => response.cards;
+getCards.injections = [
+  withPostSuccess((dispatch, response, state) => alert(`Fetched at: ${state.hearthstone.count}`))
+];
+
+const service = {
+  getCards
+};
+
+// Export your service by also specifying the reducer name and the target for each action.
+export default wrapService(service, 'hearthstone', { getCards: 'cards' });
```

Comment:
Sry about to comment this now and not before in the PR of this r-r implementation, but wouldn't it be better for this wrapper to receive only an object as a parameter, with a obj prop for each parameter that you want to pass? I think it would be a bit better for the developer's understanding and code tracking, also u forget to worry about the order of the func parameters.

Reply:
I leave here a concern. Would be a good approach to extract the reducer name as a const?

Reply:
I'm not sure about this. We may want to use a service across more than one reducer.

Reply:
> but wouldn't it be better for this wrapper to receive only an object as a parameter, with a obj prop for each parameter that you want to pass?

Would add an alternative later if feels more comfortable. By default, the idea is to 'assign' a dispatcheable service with a `reducer`. This forces us to write a `reducer` per screen, or to write them as smallest as possible. 

Also, the third parameter IMHO shouldn't be used too much, like:

```
export default wrapService(service, 'hearthstone');
```

So the targets becomes the service name: `getCards: [], getCardsLoading: false, getCardsError: null`. This approach forces us to keep a single entity for API calls in our project.

Obviously, this now has many limitations, but with this approach we point to optimize the common cases by writing less code. This approach can be used with `fetchMiddleware` or even `thunk actions` as well.

Reply:
> I leave here a concern. Would be a good approach to extract the reducer name as a const?

Yep ! This should be a constant name.

Reply:
> I'm not sure about this. We may want to use a service across more than one reducer.

The main idea is not to, but if we need it:
- We can dispatch these actions instead, wait for the response and read it using `getState()`
- We can avoid wrapping this